apis/Google.Cloud.Datastore.V1/Google.Cloud.Datastore.V1.IntegrationTests/DatastoreDbTest.cs

```diff
@@ -68,6 +69,35 @@ public void Lookup_NamespaceOnly()
             Assert.Equal("bar", (string)entity["foo"]);
         }
 
+        [Fact(Skip = "Multiple databases are available only in preview right now.")]
```

Comment:
Do you have access to check that the test passes in an environment where it's available?